metrics/mauve/README.md

```diff
@@ -41,16 +42,16 @@ from evaluate import load
 mauve = load('mauve')
 predictions = ["hello world", "goodnight moon"]
 references = ["hello world",  "goodnight moon"]
-mauve_results = mauve.compute(predictions=predictions, references=references) 
+mauve_results = mauve.compute(predictions=predictions, references=references)
 ```
 
 It also has several optional arguments:
 
 `num_buckets`: the size of the histogram to quantize P and Q. Options: `auto` (default) or an integer.
 
-`pca_max_data`: the number data points to use for PCA dimensionality reduction prior to clustering. If -1, use all the data. The default is `-1`.
+`pca_max_data`: the number of data points to use for PCA dimensionality reduction prior to clustering. If -1, use all the data. The default is `-1`.
 
-`kmeans_explained_var`: amount of variance of the data to keep in dimensionality reduction by PCA. The default is `0.9`.
+`kmeans_explained_var`: the amount of variance of the data to keep in dimensionality reduction by PCA. The default is `0.9`.
 
 `kmeans_num_redo`: number of times to redo k-means clustering (the best objective is kept). The default is `5`.
 
```

Comment:
Missing an 'Output Values' section (see e.g. [BLEU](https://github.com/huggingface/evaluate/tree/main/metrics/bleu))

Reply:
The "output values" section was already there in the initial version.